plugins/main/ptp/ptp.go

```diff
@@ -120,26 +127,33 @@ func setupContainerVeth(netns, ifName string, mtu int, pr *types.Result) (string
 	return hostVethName, err
 }
 
-func setupHostVeth(vethName string, ipConf *types.IPConfig) error {
+func setupHostVeth(vethName string, ipamResult *types.Result) error {
 	// hostVeth moved namespaces and may have a new ifindex
 	veth, err := netlink.LinkByName(vethName)
 	if err != nil {
 		return fmt.Errorf("failed to lookup %q: %v", vethName, err)
 	}
 
-	// TODO(eyakubovich): IPv6
+	ipConfig := ipamResult.IP6
+	cidrOnes := 128
```

Comment:
please declare these magic numbers as const, like so:
```
const (
    CIDROnesIPv6 = 128
    CIDROnesIPv4 = 32
)
```
Of course make sure I understood the names correctly ;-)